src/test/e2e/runner_inputs_test.go

```diff
@@ -152,4 +152,10 @@ func TestRunnerInputs(t *testing.T) {
 		require.NoError(t, err, stdOut, stdErr)
 		require.Contains(t, stdErr, "default-value")
 	})
+
```

Comment:
Can you add a test that checks for error upon calling a task with a required input without setting it via the cli flag

Reply:
The `test that direct calling of task without default values for required inputs fails` test already does that.